### DISCUSSION TITLE
Voice in frontline accessibility + iconography feedback

### DISCUSSION THREAD

**Discussion by travisterwilligar** [twilio-labs/paste#1596] [Office hours] [answered]
Based on a previous DSYS office hours meeting, we had discussed the possibility of including some of the voice specific icons in the paste library. Additionally, I'm looking to get native iOS accessibility, and general feedback if you have any.

https://www.figma.com/file/QdKFywQWu3nDKAc1mEppYe/Voice-in-Frontline?node-id=121%3A4229

**Comment by SiTaggart** [comment]
Thanks @travisterwilligar. We'll certainly do our best, though remember, we don't actually have much, if any, native mobile experience on the team officially.

Also heads up, it will only be engineers present this week, but I don't actually think that'll be too much of a problem. See you then

**Comment by serifluous** [comment, marked as answer]
Here's the recording from this office hours session: https://www.loom.com/share/e02c516f94564a68916b767db53abc2f

Password can be found in Slack.
